packages/signal-polyfill/src/wrapper.spec.ts

```diff
@@ -978,6 +978,7 @@ describe("type checks", () => {
       TypeError,
     );
     expect(Signal.subtle.Watcher.prototype.watch.call(w, s)).toBe(undefined);
+    // @ts-expect-error
```

Comment:
why expect error?

Reply:
If you look below there's a bunch more pre-existing `ts-expect-errors`. Asides from that I think it's self explanatory in these cases why all of these are present. It's specifically testing areas with incorrect types to produce TypeErrors.